hashicorp/boundary-ui
# feat: reapply storage policies to session recording

https://hashicorp.atlassian.net/browse/ICU-12230
https://hashicorp.atlassian.net/browse/ICU-12231

## Description

This PR allows the user to manually reapply the storage policy to a session recording. 

## Screenshots (if appropriate):
<img width="452" alt="Screenshot 2024-01-16 at 7 17 12 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/7a189b6c-b689-43fb-92e5-4901e26a11b2">


https://github.com/hashicorp/boundary-ui/assets/15043878/cf748587-bfae-490b-8a78-022dd7b27c5d


## How to Test



- using mirage - go to any session recording channels page click on the manage dropdown. You should be able to refresh the retain until and delete after days 



:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
